README.md

```diff
@@ -19,6 +19,9 @@ they may be customized as needed.
 | SERVICE_PRINCIPAL_CLIENT_ID            | The service principal name used to get the token credentials to connect to the APIM                | string  |
 | SERVICE_PRINCIPAL_SECRET               | The service principal secret used to get the token credentials to connect to the APIM              | string  |
 | SERVICE_PRINCIPAL_TENANT_ID            | The service principal tenant id used to get the token credentials to connect to the APIM           | string  |
+| ADB2C_CLIENT_ID                        | The service principal name used to get the token credentials to connect to the ADB2C               | string  |
```

Comment:
these are parameters relative to an application registered into the ADB2C tenant